vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -154,3 +154,320 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
         [![CustomNavigation]][CustomNavigation]
 
     [CustomNavigation]: ../../assets/user_guides/navigation/custom_navigation.png
+
+
+## Changing Selectors
+
+If you want to have a different selector for dashboard navigation you have to provide the `selector` argument of the [`Navigation`][vizro.models.Navigation] with a different selector.
+Currently available selectors are [`NavBar`][vizro.models.NavBar] and [`Accordion`][vizro.models.Accordion]. If `selector` is not defined it defaults to `Accordion`.
+
+If you want to add icons to your dashboard, you have to provide [`NavBar`][vizro.models.NavBar] model
+
+   ```
+    Dashboard(
+        pages=[page_1, page_2],
+        navigation=Navigation(
+            pages=["My first page", "My second page"]
+            selector=NavBar()
+        )
+    )
+   ```
+
+!!! example "Adding navigation icons"
```

Comment:
New comment we did not always follow, but I think its quite nice if once doesn't have to scroll for the examples on a reasonably sized screen. Could you maybe format this and the other examples such that one does get the scroll to the right because of our 120 line length I presume?